ibis/backends/base/sqlglot/compiler.py

```diff
@@ -266,18 +266,31 @@ def translate(self, op, *, params: Mapping[ir.Value, Any]) -> sge.Expression:
         def fn(node, _, **kwargs):
             result = self.visit_node(node, **kwargs)
 
-            if node is op:
+            # if it's not a relation then we don't need to do anything special
+            if not isinstance(node, ops.Relation):
```

Comment:
FWIW I prefer the previous branching because it is easier to spot unhandled or wrongly handled operation types.

Reply:
There were only ever two cases: Relations and Values. I think I prefer the dedented version slighty because the more complex code is not buried.